Zend/zend_execute_API.c

```diff
@@ -1522,7 +1522,8 @@ static void zend_set_timeout_ex(zend_long seconds, bool reset_signals) /* {{{ */
 			t_r.it_value.tv_sec = seconds;
 			t_r.it_value.tv_usec = t_r.it_interval.tv_sec = t_r.it_interval.tv_usec = 0;
 
-# if defined(__CYGWIN__) || defined(__PASE__)
+# if defined(__CYGWIN__) || defined(__PASE__) || defined(__aarch64__)
```

Comment:
Probably want to check something like `(defined(__aarch64__) && defined(__APPLE__))` instead of just `defined(__aarch64__)`?

Reply:
that s true however is it a mac arm64 issue or a macos lastest release one (there is a mention of testing on old intel mac but unsure how old the os is but I might have missed something) ? sonoma had introduced quite some "surprises" lately. Anyhow, CI for sonoma is only arm64..

Reply:
Anyway, the timer must not be changed on non-Apple aarch64.

Reply:
```suggestion
# if defined(__CYGWIN__) || defined(__PASE__) || (defined(__aarch64__) && defined(__APPLE__))
```